docs/sitemaps.rst

```diff
@@ -0,0 +1,19 @@
+Sitemaps
```

Comment:
Is this linked from anywhere? Should be in an toctree somewhere.

Reply:
Yes. I wanted to link it under `Feature Documentation` but I forgot to do it.

I will move this file under `docs/features/sitemaps.rst` and will be linked automatically on that section.